ts/features/bonus/cdc/saga/networking/handlePostRegistraBeneficiario.ts

```diff
@@ -0,0 +1,68 @@
+import { call, put } from "typed-redux-saga/macro";
+import { ActionType } from "typesafe-actions";
+import { BackendCdcClient } from "../../api/backendCdc";
+import { cdcEnrollUserToBonus } from "../../store/actions/cdcBonusRequest";
+import { getGenericError, getNetworkError } from "../../../../../utils/errors";
+import { AnniRiferimento } from "../../../../../../definitions/cdc/AnniRiferimento";
+import { ListaEsitoRichiestaPerAnno } from "../../../../../../definitions/cdc/ListaEsitoRichiestaPerAnno";
+import { CdcBonusEnrollmentOutcomeList } from "../../types/CdcBonusRequest";
+
+const convertSuccess = (
+  listaEsitoRichiestaPerAnno: ListaEsitoRichiestaPerAnno
+): CdcBonusEnrollmentOutcomeList =>
+  listaEsitoRichiestaPerAnno.listaEsitoRichiestaPerAnno.map(o => ({
+    year: o.annoRiferimento,
+    outcome: o.esitoRichiesta
+  }));
+
+export function* handlePostRegistraBeneficiario(
+  postRegistraBeneficiario: BackendCdcClient["postRegistraBeneficiario"],
+  action: ActionType<typeof cdcEnrollUserToBonus.request>
+) {
+  const requestPayload: { anniRiferimento: AnniRiferimento } = {
+    anniRiferimento: {
+      anniRif: action.payload.map(y => ({
+        anno: y.year
+      }))
+    }
+  };
+
+  try {
+    const postRegistraBeneficiarioResult = yield* call(
+      postRegistraBeneficiario,
+      requestPayload
+    );
+
+    if (postRegistraBeneficiarioResult.isRight()) {
+      if (postRegistraBeneficiarioResult.value.status === 200) {
+        yield* put(
+          cdcEnrollUserToBonus.success(
+            convertSuccess(postRegistraBeneficiarioResult.value.value)
+          )
+        );
+        return;
+      }
+      if (postRegistraBeneficiarioResult.value.status === 400) {
+        cdcEnrollUserToBonus.failure(
```

Comment:
Same as above (`yield* put`).

Reply:
Fixed in [cc3e480](https://github.com/pagopa/io-app/pull/3927/commits/cc3e48002f56fce8b0a029df9e3656518b0efca4)